catalog/README.md

```diff
@@ -0,0 +1,21 @@
+# Catalog
+
+This package holds i18n for dbaas-controller.
+
+[gotext](https://pkg.go.dev/golang.org/x/text@v0.3.3/cmd/gotext?tab=doc)
+
+To translate dbaas-controller's messages, we need to follow the next steps.
+
+- Use shared `golang.org/x/text/message.Printer` object's `Sprintf` method for messages that should be translated.
+  Do not use `Sprint` as it will not be collected on the next step.
```

Comment:
@askomorokhov did you encounter this?

Reply:
Going to remove - missed this old item.

Reply:
I just added it. But can you verify that problem?

Reply:
Oh, it was removed. Good. Yes, I met it  - but as I figure out later it was caused not by Sprint but by another error.